tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/test_formula_5_3b.py

```diff
@@ -0,0 +1,79 @@
+"""Testing formula 5.3b of NEN-EN 1992-1-1+C2:2011."""
+
+import pytest
+
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011.chapter_5_structural_analysis.formula_5_3b import Form5Dot3BTransverseForceBracedMembers
+from blueprints.validations import NegativeValueError
+
+
+class TestForm5Dot3BTransverseForceBracedMembers:
+    """Validation for formula 5.3b from NEN-EN 1992-1-1+C2:2011."""
+
```

Comment:
add integration test for using Form5Dot1Imperfections as documented in the docstrings